frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -0,0 +1,234 @@
+} from '../../../../../dev-console/src/components/NamespacedPage';
+import { createVM } from '../../../k8s/requests/vm/create/simple-create';
+import { CreateVMForm } from '../forms/create-vm-form';
+import { getTemplateOSIcon } from '../../vm-templates/os-icons';
+import { formReducer, initFormState } from '../forms/create-vm-form-reducer';
+import { useStorageClassConfigMap } from '../../../hooks/storage-class-config-map';
+import { SUPPORT_URL } from '../../../constants/vm-templates';
+import { useVmTemplatesResources } from '../hooks/use-vm-templates-resources';
+
+export const DevConsoleCreateVmForm: React.FC<RouteComponentProps> = () => {
+  const { t } = useTranslation();
+  const urlParams = new URLSearchParams(window.location.search);
+  const templateParam = urlParams.get('template');
+  const templateNs = urlParams.get('templateNs') || 'openshift';
+  const namespace = urlParams.get('namespace') || 'default';
+
+  const {
+    pods,
+    dataVolumes,
+    pvcs,
+    userTemplates,
+    baseTemplates,
+    resourcesLoaded,
+    resourcesLoadError,
+  } = useVmTemplatesResources(namespace);
+
+  const selectedTemplate = filterTemplates(userTemplates, baseTemplates)
+    .filter((tmp) => {
+      const tempSourceStatus = getTemplateSourceStatus({
+        pods,
+        pvcs,
+        dataVolumes,
+        template: tmp.variants[0],
+      });
+
+      if (isTemplateSourceError(tempSourceStatus) || !tempSourceStatus?.isReady) {
+        return false;
+      }
+      return true;
+    })
+    .find(
+      (tmp) =>
+        templateParam === tmp?.metadata?.name &&
+        tmp?.variants?.[0].metadata.namespace === templateNs,
+    );
+
+  const [state, dispatch] = React.useReducer(formReducer, initFormState(namespace));
+  const [isSubmitting, setSubmitting] = React.useState(false);
+  const [createError, setCreateError] = React.useState<string>();
+  const [scConfigMap, scLoaded, scError] = useStorageClassConfigMap();
+  const template = selectedTemplate?.variants?.[0];
+  const sourceStatus = getTemplateSourceStatus({
+    pvcs,
+    pods,
+    dataVolumes,
+    template,
+  });
+
+  return (
+    <NamespacedPage hideApplications disabled variant={NamespacedPageVariants.light}>
+      <div className="co-m-pane__body">
+        <h1 className="co-m-pane__heading">
+          {t('kubevirt-plugin~Review and create virtual machine')}
+        </h1>
+        <Helmet>
+          <title>{t('kubevirt-plugin~Virtual Machines')}</title>
+        </Helmet>
+        <StatusBox
+          data={template}
+          loaded={resourcesLoaded}
+          loadError={resourcesLoadError}
+          label="Virtual Machine Template"
+          EmptyMsg={() => (
+            <EmptyState>
+              <Title headingLevel="h4" size="lg">
+                {t('kubevirt-plugin~Error Loading Template')}
+              </Title>
+              <EmptyStateBody>
+                {t('kubevirt-plugin~Virtual machine template')}: <strong>{templateParam}</strong>{' '}
```

Comment:
I think translations are not supposed to be done like this as this usage expects an english grammar.